frontend/src/metabase/entities/fields.js

```diff
@@ -191,6 +191,14 @@ const Fields = createEntity({
         updateIn(state, [fieldId, "remappings"], (existing = []) =>
           Array.from(new Map(existing.concat(remappings))),
         ),
+      // Can't import metabase/dashboard/FETCH_CARD_DATA because of circular dependency
+      ["metabase/dashboard/FETCH_CARD_DATA"]: (state, { payload }) => {
```

Comment:
Use hard-coded action since `metabase/dashboard/FETCH_CARD_DATA` can't be imported due to cyclic dependencies.